src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import '../index.css';
```

Comment:
- [ ] Your calculator is not displayed because you are importing a non-existent `index.css` file. This should be `./calculator.css`

  ![fhfu](https://user-images.githubusercontent.com/36542986/199231088-3ade68b4-9470-44df-adfb-1df56fa9b2e7.jpg)

Reply:
ohhh, that is true, I forgot to switch to calculator.css after moving the codes from index.css to calculator.css

I will change it now